frontend/main-app/src/components/Navigation/Navigation.tsx

```diff
@@ -22,36 +22,33 @@ export const Navigation = (): React.JSX.Element => {
     return (
       <nav className={classes.navigation}>
         <UserDropdown />
-        <DropdownMenu size="small">
-          <DropdownMenu.Trigger variant="secondary" icon="true">
-            <MenuHamburgerIcon className={classes.menuIcon} />
-          </DropdownMenu.Trigger>
-          <DropdownMenu.Content>
-            <DropdownMenu.Group>
-              <DropdownMenu.Item asChild>
-                <Link href={"/dashboard"}>{t("dashboard")}</Link>
-              </DropdownMenu.Item>
-              <DropdownMenu.Item asChild>
-                <Link href="/form-builder">{t("form_builder")}</Link>
-              </DropdownMenu.Item>
-            </DropdownMenu.Group>
-          </DropdownMenu.Content>
-        </DropdownMenu>
+        {/*<DropdownMenu size="small">*/}
+        {/*  <DropdownMenu.Trigger variant="secondary" icon="true">*/}
+        {/*    <MenuHamburgerIcon className={classes.menuIcon} />*/}
+        {/*  </DropdownMenu.Trigger>*/}
+        {/*  <DropdownMenu.Content>*/}
+        {/*    <DropdownMenu.Group>*/}
+        {/*      <DropdownMenu.Item asChild>*/}
+        {/*        <Link href={"/dashboard"}>{t("dashboard")}</Link>*/}
+        {/*      </DropdownMenu.Item>*/}
+        {/*    </DropdownMenu.Group>*/}
+        {/*  </DropdownMenu.Content>*/}
+        {/*</DropdownMenu>*/}
```

Comment:
I see that we use user Dropdown, do we need these commented lines ?

Reply:
My thought was that we could use it later to display links to e.g. Login, Signup, About, etc?